teammapper-backend/src/map/services/maps.service.ts

```diff
@@ -47,27 +47,39 @@ export class MapsService {
     this.mapsRepository.update(uuid, { lastAccessed })
   }
 
-  async exportMapToClient(uuid: string): Promise<IMmpClientMap> {
+  async exportMapToClient(uuid: string): Promise<IMmpClientMap | undefined> {
     const map = await this.findMap(uuid)
     if (!map) return Promise.reject(new EntityNotFoundError("MmpMap", uuid))
```

Comment:
I would suggest to standardize our return values. E.g. when a map can't be found always return either rejected promise with error or return a fulfilled one with undefined. As you refactored it to use a fulfilled one, lets also use it here?